src/zoomifytilesource.js

```diff
@@ -0,0 +1,142 @@
+(function($) {
+
+    /**
+     * @class ZoomifyTileSource
+     * @classdesc A tilesource implementation for the zoomify format.
+     *
+     * A description of the format can be found here:
+     * https://ecommons.cornell.edu/bitstream/handle/1813/5410/Introducing_Zoomify_Image.pdf
+     *
+     * There are two ways of creating a zoomify tilesource for openseadragon
+     *
+     * 1) Supplying all necessary information in the tilesource object. A minimal example object for this method looks like this:
+     *
+     * {
+     *      type: "zoomifytileservice",
+     *      width: 1000,
+     *      height: 1000,
+     *      tilesUrl: "/test/data/zoomify/"
+     * }
+     *
+     * The tilesize if currently hardcoded to 256. The tileUrl must the the path to the image _directory_.
```

Comment:
Typo: `if` should be `is`. Also, looks like it should be `tileSize` rather than `tilesize`.

Also, why is the tileSize hardcoded to 256? Perhaps add the explanation to the comment?